### DISCUSSION TITLE
关于 Taro v3.6 Minor 版本代号的命名类别投票

### DISCUSSION THREAD

**Discussion by ZakaryCode** [NervJS/taro#12773] [Polls]
背景：从 Taro `v3.6` 开始，对该 Minor 或 Major 版本作出较大贡献的同学，将拥有对该版本的英文代号的**提名权**。最终经 Taro 技术委员会和合作者共同在提名中确认最终的版本号。希望能对贡献者起到激励作用。

根据两周前的[讨论与投票结果](https://github.com/NervJS/taro/discussions/12680)，Taro 未来的 Minor 版本代号确定为「**游戏/动漫**」，接下来我们将投票决定 v3.6 版本最终的代号。

![image](https://user-images.githubusercontent.com/24262362/199699367-1d0680a9-08f1-416c-a231-208f9633ccc0.png)

目前对 Taro v3.6 版本拥有代号命名权的同学：

- @xueshuai - 升级 PostCSS8
- @AdvancedCat - 小程序实现 History API
- @bigmeow - 优化重构 Mini CI 插件
- @robinv8  - 实现类型系统的自动化
- @zhiqingchen - RN 0.70

**Comment by AdvancedCat** [comment]
**Link（林克）**

游戏角色：《塞尔达》男主角，在海拉鲁大陆中翻山越岭，攻坚克难，勇闯天涯。
英文释意：联系，纽带。 就像 Taro 期望能够实现“一码多端”的愿景，起到跨端纽带的作用。

![](https://static.wikia.nocookie.net/zelda/images/2/29/%E3%80%8A%E5%A1%9E%E5%B0%94%E8%BE%BE%E4%BC%A0%E8%AF%B4%EF%BC%9A%E6%97%B7%E9%87%8E%E4%B9%8B%E6%81%AF%E3%80%8B%E5%BC%93%E6%89%8B%E6%9E%97%E5%85%8B.png/revision/latest/top-crop/width/360/height/360?cb=20210929161907&path-prefix=zh)

**Comment by robinv8** [comment]
Ari (阿里)，有獅子、太陽、海洋的意思，有种笼罩万物，海纳百川的意思。

虽然不是出自游戏/动漫，但想到这个的原因是，儿子在学说话，最近发现他在用 ipad 跟 siri 对话，我们教他说 siri，他不太会说，他的发音是 Ari，我打算把这个名字当成儿子的英文名。

**Comment by bigmeow** [comment]
**Overwatch (守望先锋)**

——“这个世界需要更多的英雄”
 ——“这个仓库也是。做不了世界的英雄，但可以是统一小程序的先锋”

**Comment by zhiqingchen** [comment]
Honor Of Kings
王者荣耀

**Comment by xueshuai** [comment]
Reach（致远星）
也被称作波江座 ε 行星系统 II 号行星（英文：Epsilon Eridani II），是一颗位于波江座 ε 行星系统的人类近地殖民星球。由于其距太阳系仅10.5光年，因此致远星堪称地球的门户。
2552年，致远星沦陷，整颗星球最后都被玻璃化。
2553年，战争结束一年后，UNSC 勘察致远星在战争中被破坏的情况，并着手进行环境改造。
2589年，致远星已经恢复到了可居住状态，新一批人类殖民者再次在星球上繁衍生息。
它是人类不屈和希望的象征。

**Reply by xueshuai** [reply]
![致远星](https://static.wikia.nocookie.net/halo/images/6/61/HR-PlanetReach-Bnet.jpg/revision/latest/scale-to-width-down/350?cb=20220612072229&path-prefix=zh)
